builds/unit-test.properties

```diff
@@ -0,0 +1,20 @@
+#
+# Copyright (c) 2023 OceanBase.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# OceanBase Cloud Database
+odc.ob.default.oracle.commandline=ENC@MWXHK8XjBh61FwewPZbS5N0rsHVc7yMCIrcB16SFqIESe5Dr8t47rf/AOgZwxMScCdLDXnpkyfIoANK5s2RblHgnmk11q7enoGzYb075X4T+GvxfaIUoK4VvaK7xrtm9USR8BTmb3g6Pk3fXG8YzeQ==
```

Comment:
avoid this mechanism, let's hold every sensitive value out of source code, use github action secret straightly

Reply:
Get and done!